apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.js

```diff
@@ -28,16 +28,10 @@ const Footer = ({ unsaved, readOnly }) => {
 };
 
 const DrawWidget = ({ sketchName, metadata, setHiddenInput }) => {
-  const [readOnly, setReadOnly] = React.useState(false);
   const [unsaved, setUnsaved] = React.useState(false);
   const [lib, setLib] = React.useState(null);
   const [sceneVer, setSceneVer] = React.useState(0);
 
-  // First time setup
-  React.useEffect(() => {
-    setReadOnly(metadata.read_only);
-  }, [metadata]);
-
```

Comment:
@nishanthkarthik @reteps this was flagged by https://eslint-react.xyz/docs/rules/hooks-extra-no-direct-set-state-in-use-effect. AFAICT there's no need for us to track read-only in state, but I'd appreciate a second set of eyes on this.

Reply:
I think you're right. https://github.com/PrairieLearn/PrairieLearn/blob/93a1d7899d70664db3a435b1b1cb1163c0d61483/apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py#L165 sets the field once during render. 

I don't remember why I did that but my gut feeling is that `readOnly` was dynamic at one point and the state survived through refactors.